### DISCUSSION TITLE
WebView2 Registry keys

### DISCUSSION THREAD

**Discussion by ajtruckle** [MicrosoftEdge/WebView2Feedback#4068] [Q&A]
Hi

I was using CCleaner on my PC. Why do I have all these entries in my registry?

![WebView21](https://github.com/MicrosoftEdge/WebView2Feedback/assets/7985331/4b043749-ef24-483c-8623-87119b20c2b4)
![WebView22](https://github.com/MicrosoftEdge/WebView2Feedback/assets/7985331/a6b825f1-d2a4-4df0-b538-436f2e36cfc0)

Thanks.

**Comment by Danielx64** [comment]
Tip: I would dump that cleaner program, in the days it was good, now it is not needed.

**Comment by ajtruckle** [comment]
@Danielx64  I would argue that. I only installed it today and it detected thousands of obsolete software registry entries (software that I deleted along time ago). Active control registrations that literally don't exist anymore.

**Comment by ajtruckle** [comment]
@liminzhu @victorhuangwq Insight?
